src/Core/Parser/Scene.cpp

```diff
@@ -76,6 +76,15 @@ int Raytracer::Scene::_parseCameraSetting(const libconfig::Setting &camera)
         Math::Vector3D rotation(_parseValue(rotationX), _parseValue(rotationY), _parseValue(rotationZ));
         this->_camera.setRotation(rotation);
     }
+
+    if (camera.exists("translation")) {
+        libconfig::Setting& translation = camera.lookup("translation");
+        Math::Vector3D trans(_parseValue(translation["x"]), _parseValue(translation["y"]), _parseValue(translation["z"]));
+        Math::Vector3D neworigin = this->_camera.getOrigin();
+        neworigin.translate(trans);
+        this->_camera.setOrigin(neworigin);
```

Comment:
Rename the varaible to newOrigin.
```suggestion
        Math::Vector3D newOrigin = this->_camera.getOrigin();
        newOrigin.translate(trans);
        this->_camera.setOrigin(newOrigin);
```